feature-libs/organization/administration/styles/_list.scss

```diff
@@ -1,5 +1,5 @@
 @mixin is-current-active-selection {
-  outline: solid 3px var(--cx-color-primary);
+  outline: solid 3px green;
```

Comment:
what's that green color?

Reply:
What a shame! The green color is set for testing purpose, made by me after the original commit is made. Next time before I make commit after the first code review, I will double check to avoid any testing change is pushed.